monai/networks/nets/unet.py

```diff
@@ -64,18 +64,21 @@ def __init__(
         Note: The acceptable spatial size of input data depends on the parameters of the network,
             to set appropriate spatial size, please check the tutorial for more details:
             https://github.com/Project-MONAI/tutorials/blob/master/modules/UNet_input_size_constrains.ipynb.
-            Typically, applying `resize`, `pad` or `crop` transforms can help adjust the spatial size of input data.
+            Typically, when using a stride of 2 in down / up sampling, the output dimensions are either half of the
+            input when downsampling, and twice when upsampling, so if have N numbers of layers in the network,
```

Comment:
I would change this line to "input when downsampling, or twice when upsampling. In this case with N numbers of layers in the network"

Reply:
Sure, updated in the latest commit.
Thanks for your review!